lib/drivers/docs/medtronic600.md

```diff
@@ -17,7 +17,7 @@
   - `[x]` scheduled basal
     - `[x]` basal rate intervals with a start time, duration, and rate delivered
     - `[x]` name of basal schedule on each scheduled basal rate interval
-    - `[?]` if basal schedule is a single (flat) rate all day, pump records a new basal rate interval every midnight
+    - `[x]` if basal schedule is a single (flat) rate all day, pump records a new basal rate interval every midnight
```

Comment:
Oh, does it split up a flat basal rate automatically? That's pretty cool, not many pumps do that.